glue/viewers/matplotlib/state.py

```diff
@@ -34,13 +37,69 @@ def notify(self, *args, **kwargs):
 
 VALID_WEIGHTS = ['light', 'normal', 'medium', 'semibold', 'bold', 'heavy', 'black']
 
-VALID_LOCATIONS = ['best',
+VALID_LOCATIONS = ['best (draggable)', 'best',
                    'upper right', 'upper left',
                    'lower left', 'lower right',
                    'center left', 'center right',
                    'lower center', 'upper center']
 
 
+class MatplotlibLegendState(State):
+    """The legend state"""
+
+    visible = DeferredDrawCallbackProperty(False, docstring="Whether to show the legend")
+
+    loc_and_drag = DeferredDrawSelectionCallbackProperty(0, docstring="The location of the legend in the axis")
```

Comment:
I think this would be clearer as just ``location`` as it was before - then the option can be ``draggable`` when it is draggable.

Reply:
yes,

I changed back that name.
To have the argument to pass to matplotlib API, the property 'mpl_location' need to be used now